### PR TITLE
[12.0] Add module product_expiry_stock_move_line_use_date

### PR DESCRIPTION
**Pull request by grindtildeath**
This module extends the functionality of `product_expiry` module in order to
allow users to enter a 'Best before Date' on the stock move line before the lot
creation.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@grindtildeath Is this WIP ?

**Comment by grindtildeath** [comment]
@rousseldenis It wasn't supposed to be :laughing: 

As usual there can still be some fixes coming in the next few days...

**Review by Tonow-c2c — APPROVED** [review]
LGTM

**Comment by rousseldenis** [comment]
@grindtildeath I think few days have passed :smile: What's the status of this?

**Comment by grindtildeath** [comment]
@rousseldenis I didn't had to do any other fixes since then, so I guess the module works well for the customer using it. :sweat_smile: 
IMO PR could be merged if there are enough reviews.

File: product_expiry_stock_move_line_use_date/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        context = res.get('context').copy()
+        context['show_lots_date_use'] = (
+            context.get('show_lots_text') and
+            self.product_id.display_use_date_at_lot_creation
+        )
+        res['context'] = context
+        return res
+
+
+class StockMoveLine(models.Model):
```
Comment (rousseldenis) [inline product_expiry_stock_move_line_use_date/models/stock_move.py:36]: Please split one model per file

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code erview. Minor comment

**Pull request closed without merging**